README.md

```diff
@@ -0,0 +1,155 @@
+these lists were constructed, refer to
+https://github.com/greenelab/tad_pathways. 
+
+Input this gene list directly into a
+[WebGestalt Pathway Analysis](http://webgestalt.org/) and skip to the
+[WebGestalt step](#webgestalt-pathway-analysis).
+
+#### Custom
+
+Create a comma separated file where the first row of each column names the list
+of snps below in subsequent rows. There can be many columns with variable
+length rows.
+
+E.g.: `custom_example.csv`
+
+| Group 1 | Group 2 |
+| ------- | ------- |
+| rs12345 | rs67891 |
+| rs19876 | rs54321 |
+| ...     | ...     |
+
+Then, perform the following steps:
+
+```bash
+# Map custom SNPs to genomic locations
+Rscript --vanilla scripts/build_snp_list.R \
+        --snp_file "custom_example.csv" \
+        --output_file "mapped_results.tsv"
+
+# Build TAD based genelists for each group
+python scripts/build_custom_TAD_genelist.py \
+       --snp_data_file "mapped_results.tsv" \
+       --output_file "custom_tad_genelist.tsv"
+```
+
+Skip now to the the [WebGestalt step](#webgestalt-pathway-analysis).
+
+### WebGestalt Pathway Analysis
+
+Insert either the GWAS curated genelist or a column from the custom genelist
+with the following parameters:
+
+| Parameter | Input |
+| --------- | ----- |
+| Select gene ID type | *hsapiens__gene_symbol* |
+| Enrichment Analysis | *GO Analysis* |
+| GO Slim Classification | *Yes* |
+| Reference Set | *hsapiens__genome* |
+| Statistical Method | *Hypergeometric* |
+| Multiple Test Adjustment | *BH* |
+| Significance Level | *Top10* |
+| Minimum Number of Genes for a Category | *4* |
+
+Once the analysis is complete, click `Export TSV Only` and save the file as
+`gestalt/<INSERT_TRAIT_HERE>_gestalt.tsv`. 
+
+### Curation
+
+Clean and tidy the output files and summarize into convenient lists of
+candidate genes. These genes may or may not be the nearest gene to the GWAS
+signal and will require experimental validation.
+
+```bash
+# An example for Bone Mineral Density (see `example_pipeline_bmd.sh` as well)
+
+# Process WebGestalt Output saved in `data/gestalt/bmd_gestalt.tsv`
+python scripts/parse_gestalt.py --trait 'bmd' --process
+
+# Output evidence tables
+python scripts/construct_evidence.py \
+        --trait 'bmd' \
+        --genelist 'data/gwas_catalog/Bone_mineral_density_hg19.tsv' \
+        --pathway 'skeletal system development'
+
+# Summarize evidence
+python scripts/assign_evidence_to_TADs.py \
+        --evidence 'results/bmd_gene_evidence.csv' \
+        --snps 'data/gwas_tad_genes/Bone_mineral_density_hg19_SNPs.tsv' \
+        --output_file 'results/BMD_evidence_summary.tsv'
+
+# Output venn diagram
+R --no-save --args 'results/bmd_gene_evidence.csv' \
+        'BMD' < scripts/integrative_summary.R
+```
+
+### Contact
+
+For all questions and bug reporting please file a
+[GitHub issue](https://github.com/greenelab/tad_pathways/issues)
+
+For all other questions contact Casey Greene at csgreene@mail.med.upenn.edu or
```

Comment:
i am usually not a fan of these...doesn't this open up outlook or something equivalent?